freyja/_cli.py

```diff
@@ -80,12 +123,23 @@ def demix(variants, depths, output, eps, barcodes, meta,
     # drop intra-lineage diversity naming (keeps separate barcodes)
     indexSimplified = [dfi.split('_')[0] for dfi in df_barcodes.index]
     df_barcodes = df_barcodes.loc[indexSimplified, :]
+    if lineageyml == "-1":
```

Comment:
This should only be run if we are using the `--collapsebarcodes` option, otherwise we're wasting time loading it in.